exorcist/tests/test_taskstatusdb.py

```diff
@@ -169,8 +189,72 @@ def test_get_all_tasks(self, request, fixture):
         expected = {
             'fresh_db': set(),
             'loaded_db': {('foo', TaskStatus.AVAILABLE.value, None, 0, 3),
-                          ('bar', TaskStatus.BLOCKED.value, None, 0, 3)},
+                ('bar', TaskStatus.BLOCKED.value, None, 0, 3)},
         }[fixture]
         db = request.getfixturevalue(fixture)
         tasks = set(db.get_all_tasks())
         assert tasks == expected
+
+    def test_add_task(self, fresh_db):
+        # task without prerequisites
+        fresh_db.add_task("foo", requirements=[], max_tries=3)
+        expected_foo_task = ("foo", TaskStatus.AVAILABLE.value, None, 0, 3)
```

Comment:
what does the `..., None, 0, 3)`  data here mean?

Reply:
This is the row of the `tasks` table. It returns a namedtuple (technically, an `sqla.Row`, which subclasses or wraps a namedtuple.

The entries are `(tasked, status, last_modified, tries, max_tries)`.